feature/about/src/main/res/color/version_number.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="@color/black_alpha_38" />
```

Comment:
You can do this like this 🙏 
https://github.com/droidkaigi/conference-app-2020/blob/master/corecomponent/androidcomponent/src/main/res/color/nav_drawer_item_selector.xml#L4